DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/RegistrationPrompts/Index.cshtml

```diff
@@ -14,6 +14,17 @@
   <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml"/>
 }
 
+@section NavBreadcrumbs {

```

Comment:
The more I think about it the more I wonder whether we should add the side menu to the Layout and define it similarly to the NavBreadcrumbs and the extra nav bar tabs. Then, the "double click for main content" will skip the side bar, which I think is expected + side nav vs bottom nav can be dealt with within the layout. What do you think?

Reply:
If you mean that we just define a new section and render that at the top and bottom of the layout then I don't think that will work. RenderSection in the Layout seems to be restricted to rendering unique sections - calling RenderSection twice for the same section causes the page to break. An alternative to sections might be creating something like ViewData["SideNavMenu"] = "Centre" and ViewData["CurrentPage"] = CentrePage.Administrators and performing some logic on that, which would clean up having to define both in each file. Might become fiddly trying to set up a one-quarter grid column in the layout to work with all pages, given that not all pages are going to have this side menu.

Reply:
Ah, that's fair! I think specifying it in ViewData will be too fiddly in the long run as the Layout already has quite a lot logic in it. The benefit of at least moving a side menu there is that the "Skip to main content" option will skip the side navigation bar and goes straight to content. Maybe we can just specify the side nav and bottom nav as separate sections?

Reply:
Looks like the "Skip to main content" button already goes straight to the h1 in the main content (think we noticed this before when we were working with error summaries). Do we still need separate sections?

Reply:
Ah that's actually great news! In that case, I think it's ok! 👍